examples/linux/policyfile_cookbook/habitat/plan.sh

```diff
@@ -0,0 +1,8 @@
+pkg_name=policyfile_cookbook
+pkg_origin=example-corporation
+pkg_version="0.1.0"
+pkg_maintainer="Chef"
+pkg_license=("Apache-2.0")
+pkg_scaffolding="chef/scaffolding-chef-infra"
+scaffold_policy_name="Policyfile"
+scaffold_policyfile_path="$PLAN_CONTEXT/../" # habitat/../Policyfile.rb
```

Comment:
Why not use the default `policyfiles` path instead of setting this?

Reply:
Default wants it set in a directory called `policyfiles/` next to the `plan.sh` whereas the Policyfile cookbook pattern (and the generator) just create a Policyfile.rb in the root of the cookbook.

Reply:
Sweet! Maybe remove or update the comment there? I'm not sure that path alone will make sense to folks new to the project.